stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1131,6 +1132,8 @@ public class ChatClient internal constructor(
     private fun isValidRemoteMessage(remoteMessage: RemoteMessage): Boolean =
         notifications.isValidRemoteMessage(remoteMessage)
 
+    public fun devToken(userId: String): String = ChatUtils.devToken(userId)
```

Comment:
Does it make any sense for this to be an instance method? We could also consider doing something like the new Swift SDK, where they have a `TokenProvider.development(userId)` call to create a dev token, [see here](https://github.com/GetStream/stream-chat-swift/blob/8f72ede75c64fd65e3513251d2767e968fbe63f9/Sources/StreamChat/Config/TokenProvider.swift#L23).

Reply:
It is a decision that has been taken by the iOS team, but it has not been accepted yet.

Reply:
Alright, let's go with this for now then. Feels a bit weird to make a `client.setUser(client.devToken())` kind of call, but alright.